lib/config/migrations/custom/bb-use-default-reviewers-migration.ts

```diff
@@ -0,0 +1,17 @@
+import is from '@sindresorhus/is';
+import { AbstractMigration } from '../base/abstract-migration';
+
+export class BbUseDefaultReviewersMigration extends AbstractMigration {
+  override readonly propertyName = 'bbUseDefaultReviewers';
+
+  override run(value: unknown, key: string, parentKey?: string): void {
+    const platformOptions = this.get('platformOptions') ?? {};
```

Comment:
move to condition, we only need to get the value if the condition is true (perf)